src/test/kotlin/com/mytaxi/apis/phraseapi/client/PhraseApiClientKeyTest.kt

```diff
@@ -24,43 +28,32 @@ class PhraseApiClientKeyTest {
     }
 
     @Test
-    fun `Should create a key`() {
+    fun `Should create a key with optional parameters`() {
 
         //GIVEN
         val projectId = UUID.randomUUID().toString()
         val keyName = UUID.randomUUID().toString()
         val tags = arrayListOf("abc", "def")
 
         val headers = mapOf(
-            HttpHeaders.CONTENT_TYPE to listOf(MediaType.JSON_UTF_8.toString())
+            HttpHeaders.CONTENT_TYPE to listOf(JSON_UTF_8.toString())
         )
 
-        val createKey = CreateKey(
-            name = keyName,
-            tags = tags
-        )
+        val createKey = CreateKey(name = keyName, tags = tags, description = "desc", plural = false)
 
         val keyJSON = Gson().toJson(createKey)
 
-        val response = Response.create(
-            HttpStatus.SC_CREATED,
-            "OK",
-            headers,
-            keyJSON,
-            StandardCharsets.UTF_8
-        )
+        val response = Response.create(HttpStatus.SC_CREATED, "OK", headers, keyJSON, StandardCharsets.UTF_8)
 
-        Mockito.`when`(client.createKey(
+        `when`(client.createKey(
```

Comment:
Could you change the `when` to `on`?

`import org.mockito.Mockito.`when` as on`

Reply:
Done.